simtools/applications/simulate_prod.py

```diff
@@ -335,6 +344,34 @@ def main():
 
     simulator.simulate()
 
+    logger.info(
+        f"Production run is complete for primary {shower_configs['primary']} showers "
+        f"coming from {shower_configs['phi']} azimuth and zenith angle of "
+        f"{shower_configs['zenith']} at the {args_dict['site']} site, "
+        f"using the {array_configs['model_version']} telescope model."
+    )
+
+    if args_dict["pack_for_grid_register"]:
+        logger.info("Packing the output files for registering on the grid")
+        output_files = simulator.get_list_of_output_files()
+        log_files = simulator.get_list_of_log_files()
+        histogram_files = simulator.get_list_of_histogram_files()
+        tar_file_name = Path(log_files[0]).name.replace("log.gz", "log_hist.tar.gz")
```

Comment:
I guess I do not need to know the logic of this renaming.

Reply:
This is because we tar together the log and hist files and upload to the grid that one tarball as the log of the job instead of uploading them separately.